deepreg/train.py

```diff
@@ -61,8 +61,9 @@ def build_config(
 def train(
     gpu: str,
     config_path: Union[str, List[str]],
-    gpu_allow_growth: bool,
     ckpt_path: str,
+    num_workers: int = 1,
```

Comment:
I would expect that num_workers is also used for model.fit if we are using the same name here... Which doesn't appear to be the case.

Reply:
![image](https://user-images.githubusercontent.com/4066929/114621435-99923e80-9ca4-11eb-914f-bcafa29b4759.png)
We are using tf.data therefore this argument is not used